### PR TITLE
ics27 improvements requirements doc

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

closes: #XXXX

### Commit Message / Changelog Entry

```bash
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/CONTRIBUTING.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/10-structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/CONTRIBUTING.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [x] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by damiannolan — APPROVED** [review]
Nice!

File: docs/requirements/ics27-v2-requirements.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- Make controller functionality integration in a chain easier.
+- Introduce a message server in the controller submodule that exposes APIs for interchain account registration and control.
+- Once application callbacks are implemented (via ADR 008) deprecate the APIs introduced in ibc-go v3.0.0.
+
+## Scope
+
+| Features  | Release |
+| --------- | ------- |
+| Register interchain accounts and send transactions to host chain via message passing. | v2 |
+| Support application callbacks with message server in controller submodule (requires ADR 008). | v3 |
```
[the remark this answers is not in the document]
Comment (crodriguezvega) [reply docs/requirements/ics27-v2-requirements.md:25]: Thank you, @damiannolan. I am also not sure yet how to document this, so I went for an "easy" solution. I like your proposal, so let me think about it a bit.

[the remark this answers is not in the document]
Comment (colin-axner) [reply docs/requirements/ics27-v2-requirements.md:25]: I agree with damian here. Might be nice just to use the ibc-go release version for clarity.

**Review by colin-axner — APPROVED** [review]
Nice work! Very clean/readable

**Comment by damiannolan** [comment]
Can we merge this @crodriguezvega?

**Comment by crodriguezvega** [comment]
Thank you for the reviews! I added the ibc-go release version and I will merge it.

**Pull request merged**